client/config/users.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package config
+
+import "github.com/hashicorp/nomad/nomad/structs/config"
```

Comment:
```suggestion
package config

import sconfig "github.com/hashicorp/nomad/nomad/structs/config"
```

Minor nit-picking since you can't reference you own package, so feel free to ignore this, but since there are some structs with the same name it could be useful to highlight which `config` is being used.